docs/5.management/1.account.md

```diff
@@ -0,0 +1,202 @@
+nodes and could be difficult (if not impossible in some cases) to make a
+migration from one node to another.
+
+#### What are the default quotas?
+
+By default, each user has assigned a minimum number of resources that
+can be used in the development of its solution. This quota is comprised
+of the following values:
+
+| **VM Instances** | **vCPUs** | **Hard Disk** | **Memory** | **Public IPs** |    **Routers** | **Networks** | 
+| --- | --- | --- | --- | --- | --- | --- | 
+| 2 | 2 | 40 Gb | 4096 Mb | 1 | 0 | 0 |
+
+#### What are the default maximum quotas?
+
+If the project required more resources, the users have the possibility
+to explain it and the default quotas can be extended to a maximum
+default quotas that is comprised of the following values:
+
+| **VM Instances** | **vCPUs** | **Hard Disk** | **Memory** | **Public IPs** |    **Routers** | **Networks** | 
+| --- | --- | --- | --- | --- | --- | --- | 
+| 5  | 10 | 100 Gb | 10240 Mb | 1 | 1 | 1 |
+
+Keep in mind that in order that users can obtain more resources, it is
+mandatory that they provide the corresponding justification why they
+need more resources. Exceptionally, if users need more resources, after
+a clear justification the FIWARE Lab nodes can provide extra resources
+to cover them.
+
+### Resource assignment and account upgrade by FIWARE Lab administrators
+
+This section describes how FIWARE Lab Node operators should process
+incoming requests. The process is simple and depicted below.
+
+Once, for example, a coach assigns you a ticket, you start progressing
+it, each comment will be notified to the end-user that will be in this
+way able to provide you further information. For resource
+assignment/negotiation the guide in Section [Quotas and Flavours]()
+should be considered.
+
+![Account upgrade workflow.](image15.png)
+
+If your node is not able to complete the assignment, we suggest
+negotiating the assignment of those resources in other FIWARE Lab node,
+taking into consideration user’s requirements and predefined
+accelerator's program to FIWARE Lab nodes mapping. Of course, you should
+inform users about this migration assignment to confirm that it is ok
+for them.
+
+#### How to assign resources to a new community user
+
+To manage user information, you have to use the Horizon-based Cloud
+Portal interface at
+[*https://cloud.lab.fiware.org*](https://cloud.lab.fiware.org). In this
+web site, you can to click on the button *Request Community Account* and
+you will see a dialog box requesting the information required to set up
+a community account. Once the user has filled all the information and
+submitted it, the process will start with the creation of the
+corresponding Jira ticket. The next steps that every FIWARE Lab node has
+to follow is the following:
+
+1.  Start progressing the ticket as soon as you receive it.
+
+1.  Check the requested resources and comment to inform the user about
+decision on resources, or additional information you need.
```

Comment:
do not break the line because it produce an error in the travis:

docs/5.management/1.account.md: 128: MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "decision on resources, or addi..."]